### PR TITLE
Feature/uchv 43 complaint category

### PR DESCRIPTION
**Pull request by gxxm**
* added the fields to step in route config
* added 2 forks to this step
* added journey header translation
* added field translations / validation
* added acceptance tests for this step
* lint changes

### CODE REVIEW

File: apps/complaints/index.js [hunk 1]
```diff
@@ -55,6 +55,39 @@ module.exports = {
...
+      next: '/has-reference',
+      forks: [{
+        target: '/has-complaint-reference',
+        condition: {
+          field: 'complaint-type',
+          value: 'previous'
+        }
+      }, {
+          target: '/incident-where',
+          condition(req) {
```
Comment (joefitter) [inline apps/complaints/index.js]: extra paren in this function, also I'd only grab the value once:
```
const type = req.form.values['complaint-type'];
return type === 'staff' || type === 'appointment';
```

File: apps/complaints/index.js [hunk 2]
```diff
@@ -55,6 +55,39 @@ module.exports = {
...
+            return (req.form.values['complaint-type'] === 'staff' ||
+                    req.form.values['complaint-type'] === 'appointment');
+          }
+        }],
+      fields: ['complaint-type'],
+      locals: {
+        section: 'complaint-details'
+      }
+    },
+    '/incident-where': {
```
Comment (joefitter) [inline apps/complaints/index.js]: Is this what we want the URL to be?

**Comment by bendiggle** [comment]
LGTM!

**Comment by joefitter** [comment]
LGTM

**Pull request merged**